pkg/neco/cmd/config_set.go

```diff
@@ -142,6 +144,16 @@ Possible keys are:
 					return errors.New("invalid URL")
 				}
 				return st.PutNodeProxy(ctx, value)
+			case "external-ip-address-block":
+				value = args[1]
+				ip, _, err := net.ParseCIDR(value)
+				if err != nil {
+					return err
+				}
+				if ip.To4() == nil {
+					return errors.New("not IPv4 addr: " + value)
+				}
+				return st.PutExternalIPAddressBlock(ctx, value)
```

Comment:
I think it is safer to serialize the parsed data.
```suggestion
				ip, block, err := net.ParseCIDR(value)
				if err != nil {
					return err
				}
				if ip.To4() == nil {
					return errors.New("not IPv4 addr: " + value)
				}
				return st.PutExternalIPAddressBlock(ctx, block.String())
```